sass/components/_breadcrumbs.scss

```diff
@@ -0,0 +1,9 @@
+.breadcrumbs li::after {
```

Comment:
why do u prefer this over putting the `/` in the markup?

if it's because of accessibility concerns, i think there are classes / attributes u can add so that screen readers ignore it